.github/workflows/weekly.yml

```
@@ -4,24 +4,23 @@ on:
   workflow_dispatch: # run on request (no need for PR)
   schedule:
     # every 7PM on Sunday
```

```suggestion
    # every 12AM on Sunday
```
It also needs to be modified